### PR TITLE
Test ssh connection concurrency

### PR DESCRIPTION
**Pull request by dagguh**

### CODE REVIEW

File: src/test/kotlin/com/atlassian/performance/tools/ssh/api/SshConnectionTest.kt [hunk 1]
```diff
@@ -10,8 +14,47 @@ class SshConnectionTest {
...
-            Assert.assertEquals(sshResult.output, "test\n")
+            assertThat(sshResult.isSuccessful()).isTrue()
+            assertThat(sshResult.output).isEqualTo("test\n")
         }
     }
+
+    @Test
+    fun shouldBeConcurrent() {
+        // given
+        val connectionConcurrency = 256
```
Comment (dagguh) [inline src/test/kotlin/com/atlassian/performance/tools/ssh/api/SshConnectionTest.kt:25]: @pbruski 40 parallel VU nodes should not be a problem for `newConnection`

**Comment by mzyromski-atlassian** [comment]
@dagguh Do you want to push this test given that it didn't seem to detect the #7 problem?

**Comment by dagguh** [comment]
The test doesn't affect the lib consumers problems, so it's not valuable. I'm gonna miss AssertJ and Gradle Scans, so next time I'll cherry-pick those commits.

File: src/test/kotlin/com/atlassian/performance/tools/ssh/api/SshConnectionTest.kt (hunk 1 above)
Comment (dagguh) [reply src/test/kotlin/com/atlassian/performance/tools/ssh/api/SshConnectionTest.kt:25]: So this didn't detect https://github.com/atlassian/ssh/pull/7
Perhaps it's about concurrent connections to different hosts?

**Pull request closed without merging**